### DISCUSSION TITLE
any one created a boilerplate for vupress just like the vuepress docs looks like?

### DISCUSSION THREAD

**Discussion by JenuelDev** [vuepress/core#350] [Q&A] [answered]
anyone created a boilerplate for vuepress just like the vuepress docs look like?

**Comment by fenghan34** [comment, marked as answer]
Perhaps the [vuepress docs](https://github.com/vuepress/vuepress-next/tree/main/docs) is the best boilerplate right now. #309

**Reply by JenuelDev** [reply]
Are we allowed to use it? thanks :) I'm going to use it now.

**Reply by fenghan34** [reply]
absolutely, haha.
